### PR TITLE
Crashlytics disabled for debug mode and orientations locked for Android app to Portrait (Bug Fix)

### PR DESCRIPTION
**Pull request by abhishek11008**

### CODE REVIEW

File: app/src/main/java/org/merakilearn/datasource/ApplicationRepo.kt [hunk 1]
```diff
@@ -81,6 +84,14 @@ class ApplicationRepo(
 
     suspend fun fetchClassData(classId: Int): Classes? {
         return try {
+
+            // To disable crashylytics in Debug application mode
+            if(BuildConfig.DEBUG) {
```
Comment (suhaibroomy) [inline app/src/main/java/org/merakilearn/datasource/ApplicationRepo.kt:89]: This should be done in Application class.

**Review by suhaibroomy — CHANGES_REQUESTED** [review]
Let's try to keep PRs separate for different issues. It will enable us to revert that commit in case we face issues with some fixes

File: app/src/main/java/org/merakilearn/datasource/ApplicationRepo.kt (hunk 1 above)
Comment (abhishek11008) [reply app/src/main/java/org/merakilearn/datasource/ApplicationRepo.kt:89]: got it, I'll create new PR for this.

Comment (abhishek11008) [reply app/src/main/java/org/merakilearn/datasource/ApplicationRepo.kt]: This is done now @suhaibroomy, You can pick latest commit. 

But not in new PR, I'll take care from next time separate work, separate PRs.

**Review by suhaibroomy — APPROVED** [review]

**Pull request merged**